src/coreclr/gc/gc.cpp

```diff
@@ -2172,6 +2172,13 @@ size_t      gc_heap::g_bpromoted;
 
 #endif //MULTIPLE_HEAPS
 
+size_t      gc_heap::card_table_element_layout[total_bookkeeping_elements + 1];
+#ifdef USE_REGIONS
+uint8_t*    gc_heap::bookkeeping_data = nullptr;
+uint8_t*    gc_heap::bookkeeping_data_committed = nullptr;
```

Comment:
when I see a name like bookkeeping_data, my guess would be this is where the bookkeep data is. but this is where start of the range that's covered by the bookkeeping data. perhaps we should call these bookkeeping_covered_start/bookkeeping_covered_committed? feel free to suggest something better.